stdlib/runtime/gc.gr

```diff
@@ -45,6 +45,12 @@ export let utoa32Buffered = box((a: WasmI32, b: WasmI32, c: WasmI32) => void)
 
 let mut _DEBUG = false
 
+let mut _DEBUG_HOOK_INCREF = 0n
+let mut _DEBUG_HOOK_DECREF = 0n
+let mut _DEBUG_HOOK_MALLOC = 0n
+let mut _DEBUG_HOOK_FREE = 0n
+let mut _DEBUG_HOOK_RUNNING = false
```

Comment:
It's not obvious to me that the `_DEBUG` value above doesn't have anything to do with these values. I wonder if there's any renaming we can do to make that more clear.

Reply:
Or maybe just move it to the commented code below? Currently it's never used, its for the debugging functions that are all commented out. I agree though that leak checker code can get mixed up with these.